site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -501,6 +501,13 @@ include::../shared/_advanced-edition-blurb-api.adoc[]
 +
 :advanced_feature!:
 
+[field]#sourceApplicationId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.43.0#::
+The optional Id of an existing Application from which a copy will be made.
++
+A unique [field]#application.name# is required. If present, the [field]#application.id# value will used for the new Application. If no Id is specified, a secure random UUID will be generated and used.
++
+The Oauth configuration is cleared.  Roles are copied with new ids. All other values will be copied from the source Application to the new Application.
```

Comment:
This is not accurate, only the  `oauthConfiguration.clientSecret` is cleared. 

Maybe we should say something like:

> The `oauthConfiguration.clientSecret` and each role Id will be cleared so new values can be generated. All other values will be copied from the source Application.